Docs.html

```
@@ -1,1503 +1,1320 @@
+      <div class="attribute-section">
+        <h3>
+          <span class="class-name">mouse</span>
+        </h3>
+        <p>Enables an element to work with DomPipe.js attributes on specified events.</p>
+        <div class="example-container">
+          <div class="example-title">Example</div>
+          <div class="example-code">&lt;div id="hoverElement" class="mouse" event="mouseover;mouseleave" ajax="hover-data.json" insert="hoverResult"&gt;Hover over me&lt;/div&gt;</div>
+          <div class="example-output">
+            <dotp>
+              <div id="hoverElement" style="padding: 10px; background-color: #f5f5f5; display: inline-block; cursor: pointer;">Hover over me</div>
+              <div id="hoverResult" style="margin-top: 10px; padding: 10px; border: 1px solid #ddd;">Hover data will appear here</div>
```

HTMLHint (1.9.2)

[error] The id value [ hoverResult ] must be unique.

(id-unique)